client/web/src/user/settings/auth/UserSettingsSecurityPage.tsx

```diff
@@ -203,98 +187,90 @@ export const UserSettingsSecurityPage: React.FunctionComponent<React.PropsWithCh
-                                        value={oldPassword}
-                                        onChange={onOldPasswordFieldChange}
-                                        disabled={loading}
-                                        id="oldPassword"
-                                        name="oldPassword"
-                                        aria-label="old password"
-                                        placeholder=" "
-                                        autoComplete="current-password"
-                                    />
-                                </div>
-                            )}
-
-                            <div className="form-group">
-                                <Label htmlFor="newPassword">New password</Label>
-                                <PasswordInput
-                                    value={newPassword}
-                                    onChange={onNewPasswordFieldChange}
-                                    disabled={loading}
-                                    id="newPassword"
-                                    name="newPassword"
-                                    aria-label="new password"
-                                    minLength={window.context.authMinPasswordLength}
-                                    placeholder=" "
-                                    autoComplete="new-password"
-                                />
-                                <small className="form-help text-muted">
-                                    {getPasswordRequirements(window.context)}
-                                </small>
-                            </div>
-                            <div className="form-group">
-                                <Label htmlFor="newPasswordConfirmation">Confirm new password</Label>
-                                <PasswordInput
-                                    value={newPasswordConfirmation}
-                                    onChange={onNewPasswordConfirmationFieldChange}
-                                    disabled={loading}
-                                    id="newPasswordConfirmation"
-                                    name="newPasswordConfirmation"
-                                    aria-label="new password confirmation"
-                                    placeholder=" "
-                                    minLength={window.context.authMinPasswordLength}
-                                    inputRef={setNewPasswordConfirmationField}
-                                    autoComplete="new-password"
-                                />
-                            </div>
-                            <Button
-                                className="user-settings-password-page__button"
-                                type="submit"
+            <hr className="my-4" />
+            <H3 className="mb-3">Password</H3>
+            <Container>
+                <Form onSubmit={handleSubmit}>
```

Comment:
I would be more specific here and specify the action in the header, similar as we do in the button at the end of the form.
```suggestion
            <H3 className="mb-3">{props.user.builtinAuth ? 'Update ' : 'Create '}Password</H3>
            <Container>
                <Form onSubmit={handleSubmit}>
```

I would also add a description to this section, because on the screenshot it looks weird. E.g. something like (to be confirmed by @ryphil or someone with better language skills than me):
```
<p>Create a password for this user account on the <a href="docs">built-in username/password authentication.</a></p>
```
And a similar one for the password change:
```
<p>Change password for this user account on the <a href="docs">built-in username/password authentication.</a></p>
```

Otherwise users might be confused as to which password they are going to create/change.